assets/src/editor/events/tickets/editor-ticket/price-calculator/hooks/use-base-price.js

```diff
@@ -8,13 +8,11 @@ const { BASE_PRICE_TYPES } = priceTypeModel;
 
 /**
  * @function
- * @param {number|string} ticketId
- * @param {Object} formData
- * @param {BaseEntity[]} prices,
- * @return {Array} price modifiers
+ * @param {Array} prices
+ * @return {Object} A price
  */
-const useBasePrice = useMemo( ( prices ) => prices.find(
+const useBasePrice = ( prices ) => useMemo( () => prices.find(
 	( price ) => price.PRT_ID === BASE_PRICE_TYPES.BASE_PRICE
-) );
+), [ prices ] );
```

Comment:
doesn't this run the risk of memoization issues?
previously the prices were always passed in and therefore were always "fresh"

Reply:
No, it doesn't. The reason for that is the base price entity for a ticket is always going to be the same. Technically it just needs to run once.